packages/nextjs/index.ts

```diff
@@ -61,7 +61,7 @@ export function createEnv<
 >(opts: Options<TServer, TClient, TShared>) {
   const client = typeof opts.client === "object" ? opts.client : {};
   const server = typeof opts.server === "object" ? opts.server : {};
-  const shared = typeof opts.shared === "object" ? opts.shared : {};
```

Comment:
this begs the question why are we doing this check for client/server ? the core package handles that already 🤔

Reply:
cc @chungweileong94 ?